flow/flow.go

```diff
@@ -842,6 +937,7 @@ func SetHandler(IN *Flow, handleFunction HandleFunction, context UserContext) er
 // and sent further in the same flow.
 func SetVectorHandler(IN *Flow, vectorHandleFunction VectorHandleFunction, context UserContext) error {
 	handle := makeHandler(nil, vectorHandleFunction)
+	handle.name = "vector handle"
```

Comment:
I don't understand magic with names of functions inside segment. Why do you need them?